docs/source/install/docker-compose.rst

```diff
@@ -0,0 +1,275 @@
+
+
+To trigger data ingestion in the registry, you need to run client side commands (which use docker compose as well internally)
+
+================================================ ==================================================== ==============================================
+ Command prototype                                 Description                                          Prerequisites
+
+                                                                                                        (profiles to start before this)
+
+================================================ ==================================================== ==============================================
+ pds-batch-loader.sh  <harvest job confg file>     Executes the Standalone Harvest client-side tool.    The `pds-core-registry` server-side profile
+                                                   This tool is recommended for small data sets of      must be up and running
+                                                   up to 10,000 PDS4 labels.
+ pds-service-loader.sh <harvest job confg file>    Executes the Scalable Harvest client-side tool.      The `pds-service-loader` server-side profile
+                                                   This tool is   recommended for larger data sets of   must be up and running
+                                                   over 10,000 PDS4 labels.
+================================================ ==================================================== ==============================================
+
+
+Common deployment scenarii
+****************************
+
+
+Core Registry with Standalone Harvest
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As explained above, the Standalone Harvest simple batch tool, which is suitable to process small data sets of
+up to 10,000 PDS4 labels.
+
+You can execute the following instructions to deploy the registry and load small amount of data.
+
+Deploying the Sever-side Components of Standalone Harvest
+----------------------------------------------------------
+
+1) Open a new terminal and change the current working directory to the ``<REGISTRY_ROOT>/docker`` directory.
+
+2) Start opensearch and the PDS search API as follows. ::
+
+    docker compose --profile=pds-core-registry  up
+
+3) Wait for the following log messages in the terminal. ::
+
+    docker-registry-api-1        |   .   ____          _            __ _ _
+    docker-registry-api-1        |  /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
+    docker-registry-api-1        | ( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
+    docker-registry-api-1        |  \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
+    docker-registry-api-1        |   '  |____| .__|_| |_|_| |_\__, | / / / /
+    docker-registry-api-1        |  =========|_|==============|___/=/_/_/_/
+    docker-registry-api-1        |  :: Spring Boot ::        (v2.3.1.RELEASE)
+    docker-registry-api-1        |
+
+Run Standalone Harvest to load some PDS4 products
```

Comment:
[Title Case ](https://en.wikipedia.org/wiki/Title_case)Suggestion: **Run Standalone Harvest to Load Some PDS4 Products**

Reply:
Done